### PR TITLE
Make qdm ids

### PR DESCRIPTION
**Pull request by daco101**
This PR changes cqm-converter so that hds record data elements ids are preserved and converted into qdm ids.
- [ ] Update gemspec to point to new models, and remove branch reference in Gemfile before merge

Pull requests into cqm-converter require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.

**Submitter:**
- [x] This pull request describes why these changes were made.
- [x] Internal ticket for this PR: https://jira.mitre.org/browse/BONNIE-1667
- [x] Internal ticket links to this PR
- [x] Code diff has been done and been reviewed
- [x] Tests are included and test edge cases
- [x] Tests have been run locally and pass
- [x] If applicable, the library version number in `cqm-converter.gemspec` has been updated NA

**Bonnie Reviewer:**

Name: Bonnie Group Review (@mayerm94 @hossenlopp @losborne @jbradl11)
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

**Cypress Reviewer:**

Name: @dczulada 
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -2,6 +2,13 @@ source 'https://rubygems.org'
 
 git_source(:github) { |repo_name| "https://github.com/#{repo_name}" }
 
-gem 'cql_qdm_patientapi', github: 'projecttacoma/cql_qdm_patientapi', branch: 'bonnie-v2.3'
+gem 'cql_qdm_patientapi', github: 'projecttacoma/cql_qdm_patientapi', branch: 'master'
+
+# below can be comented out to use default
+gem 'cqm-models', github: 'projecttacoma/cqm-models', branch: 'datatype_id_use_schema'
+
+group :test, :development do
```
Comment (holmesie) [inline Gemfile]: I made a comment about this same thing in the corresponding PR in cqm-models. This should be in the gemspec, if this additional dependency makes sense to have.
